### ISSUE TITLE
Provide option to render alphabets in Unicode

### ISSUE THREAD

**Issue by xaltsc** [KaTeX/KaTeX#3482]
**Describe the solution you'd like:**
Currently, KaTeX renders alphabets such as \mathfrak, \mathscr, \mathcal as the basic latin plane with a font set to get the wanted output.

However, most of these alphabets are present in Unicode, and some fonts, such as Julia Mono, have support for them. The way KaTeX currently handles fonts prevents the use of other fonts than the provided AMS-like one.

**Link to or name of a (La)TeX package that provides the same feature:**
Not applicable.

**Describe alternatives you've considered:**
Not applicable. An alternative would be equivalent to a PR.

**Comment by ronkok** [comment]
Yes, KaTeX is tightly bound to the KaTeX fonts. What that gives KaTeX is pre-knowledge of the metrics of every glyph. That pre-knowledge enables KaTeX do all its layout in one pass, even when rendering server-side. 

To generate HTML with other fonts would require a major re-write of KaTeX.

KaTeX also generates MathML. It can use any math font and it will become usable if Google ever finishes their [MathML in Chromium](https://mathml.igalia.com/news/) project and Safari cleans up their MathML a little. Maybe someday.
